src/lib/assets/rbac.ts

```diff
@@ -2,35 +2,65 @@
 // SPDX-FileCopyrightText: 2023-Present The Pepr Authors
 
 import { kind } from "kubernetes-fluent-client";
+import { PolicyRule } from "kubernetes-fluent-client/dist/upstream";
 import { CapabilityExport } from "../types";
 import { createRBACMap } from "../helpers";
+
 /**
- * Grants the controller access to cluster resources beyond the mutating webhook.
+ * Creates a Kubernetes ClusterRole based on capabilities and optional custom RBAC rules.
  *
- * @todo: should dynamically generate this based on resources used by the module. will also need to explore how this should work for multiple modules.
- * @returns
+ * @todo: should dynamically generate this based on resources used by the module. will also need to explore how this
```

Comment:
Lingering TODO comment. Either fix it or write up an issue in the backlog so we don't lose track

Reply:
Removed TODO